api/rosetta/balance.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2021 Alvalor S.A.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package rosetta
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+
+	"github.com/optakt/flow-dps/models/identifier"
+	"github.com/optakt/flow-dps/models/rosetta"
+)
+
+type BalanceRequest struct {
+	NetworkID  identifier.Network    `json:"network_identifier"`
```

Comment:
Now that there's `models/rosetta` - do you think it makes sense to merge the `identifier` package into that one? It seems like it's also only used for rosetta

Reply:
No, I don't think it makes sense to merge them, because then it would have to be `rosetta.NetworkID` instead of `identifier.Network`. I think IDs are distinct from entities.

Reply:
Are they still rosetta specific identifiers? For instance, is currency being `symbol` + `decimals`, with their explicit JSON tags be used in general, or are they specific to the rosetta context?

Reply:
The identifiers are all Rosetta specific.